Dependencies/Il2CppAssemblyGenerator/Core.cs

```diff
@@ -2,11 +2,11 @@
 using System.IO;
 using System.Net;
 using MelonLoader.Il2CppAssemblyGenerator.Packages;
+using MelonLoader.Modules;
 
 namespace MelonLoader.Il2CppAssemblyGenerator
 {
-
-	internal static class Core
+	internal class Core : MelonModule
```

Comment:
This line starts with a tab while the rest start with spaces

Reply:
... you fr

Reply:
https://github.com/LavaGang/MelonLoader/pull/197/commits/9aece8a968fd9dd195cde12bf732747f3d65204d